Sources/MongoSwift/BSON/BSONEncoder.swift

```diff
@@ -185,21 +185,21 @@ internal struct _BsonEncodingStorage {
         return array
     }
 
-    fileprivate mutating func push(container: BsonValue?) {
+    fileprivate mutating func push(container: BSONValue?) {
         self.containers.append(container)
     }
 
-    fileprivate mutating func popContainer() -> BsonValue? {
+    fileprivate mutating func popContainer() -> BSONValue? {
         precondition(self.containers.count > 0, "Empty container stack.")
         return self.containers.popLast()!
     }
 }
 
-/// _BsonReferencingEncoder is a special subclass of _BsonEncoder which has its own storage, but references the 
+/// _BSONReferencingEncoder is a special subclass of _BSONEncoder which has its own storage, but references the 
```

Comment:
Unrelated to the change, but we can probably add backticks around these class references for consistency:

```suggestion
/// `_BSONReferencingEncoder` is a special subclass of `_BSONEncoder` which has its own storage, but references the 
```